lib/states/ks/wichita.R

```diff
@@ -58,14 +58,22 @@ clean <- function(d, helpers) {
         "pedestrian",
         "vehicular"
       ),
+      # NOTE: some of the date info in the new 2018-2020 data
+      # has what appears to be some sort of timezone info at the end
+      # of the string - e.g., "2018/01/01 10:48:00+00". All 
+      # entries with this format have "+00" at the end, which we remove 
+      # for processing
+      citation_date_time = sub("\\+.*", "", citation_date_time), 
```

Comment:
I'd add a "$" at the end of the regex, just in case, but no big deal.

Reply:
good suggestion!